commons/commons-util/src/main/kotlin/ebi/ac/uk/util/file/PoiExt.kt

```diff
@@ -5,20 +5,19 @@ import org.apache.poi.ss.usermodel.Cell
 import org.apache.poi.ss.usermodel.CellType.NUMERIC
 import org.apache.poi.ss.usermodel.Row
 import org.apache.poi.ss.usermodel.Sheet
-import java.util.LinkedList
 
 /**
  * Retrieves the sheet representation as a list of tsv separated string List. Note that as stream reader ignored empty
  * rows they are completed with emtpy values.
  */
 fun Sheet.asTsvList(): List<String> {
-    val elements = LinkedList<String>()
+    val elements = mutableListOf<String>()
```

Comment:
mutable list in kotlin is by default a ArrayList which performance for remove operations is terrible. (as backing array need to be re created)

Reply:
Reverted